### ISSUE TITLE
Creating Survey

### ISSUE THREAD

**Issue by adriatic** [monterey-framework/monterey#326]
## @JeroenVinke 
You get the technical tasks

### Find the survey that:

1. Can be hosted in Wordpress
2. Forces authentication using Github account
3. Has no limit on number of users that respond to the survey
4. Allow hyperlinks in questions

_Note: Google forms could be this solution if it has the support for item 2. above_

***

## @adriatic
You get the marketing tasks

1. Write the blog (http://blog.aureliatools.com/2016/09/04/introducing-monterey/)
2. Define the set of questions for the above blog - using the currently existing survey as the model
3. Write the .md file for Rob's blog post.

***

**Comment by adriatic** [comment]
Please check http://blog.aureliatools.com/

![image](https://cloud.githubusercontent.com/assets/2712405/18647640/f004ed78-7e84-11e6-89f7-b4e3ce616cae.png)

and click on the start button. Go through the survey and report back here what would you keep / change or add.

I believe that this is sufficiently good (assuming proper set of questions, which I will slightly expand to include some general comments, ask for the GitHub ID etc.).

While waiting for your reaction, I will be writing the actual blog's content (under a different name)

**Comment by Thanood** [comment]
I like it 👍 
I'll try it again tomorrow morning and try to find an idea for "Option 3" (last question).. 😉

**Comment by adriatic** [comment]
Nothing form you since - hope you did not run into some killer bug 😏 

In the mean time, I created a **[draft version](http://blog.aureliatools.com/2016/09/19/survey-test/)** (will deal with formatting and styling issue tomorrow). The actual blog text might be good enough and the survey questions could be close to final (let me know if you have a different opinion)

**Comment by adriatic** [comment]
> I'll try it again tomorrow morning and try to find an idea for "Option 3" (last question)

This was me forgetting to remove the third option in it's default form. Of course, do   tell me what would you like to add / change  / remove as we will have at least half a day tomorrow for getting this nailed

**Comment by Thanood** [comment]
There's a typo:
> If your answer on the above question was "prefer other fatures", tell us more

"prefer other features".

I somehow stumbled at this one:

> Please share your GitHub account ID with us, so we can stay in touch and invite you to our chat rooms

I'm not really sure why, just a gut feeling.
Maybe rephrase it "If you'd like us to get in touch with you, leave your GitHub ID"

Or remove the question since users will find the chat room anyway (from the "support" button).

**Comment by JeroenVinke** [comment]
@Thanood the idea behind asking for the github account ID is so we can filter out multiple responses from a single user.

@adriatic my feedback on the blog post:
- We can mention that we have a Linux installer, but even though @vegarringdal has been able to open this a few weeks back, we shouldn't give any guarantees 😄 
- We should add info on how to install monterey, right from the blog post (after the survey seems best). The three [buttons](https://github.com/monterey-framework/monterey#installing-monterey) are HTML, so you can just copy them into the blog post
- The screenshot of Monterey in the survey looks very compressed, to a point where you can hardly see what the text in the screenshot says. We should move this screenshot outside of the survey, as a picture says more than a thousand words 😄

**Comment by adriatic** [comment]
> We should add info on how to install monterey ...

Yes, I like this and will add these buttons (other suggestions are already in place)

**Comment by JeroenVinke** [comment]
> Added later: I did not place the install buttons to the blog. They were not laid out next to each other , but rather in three successive rows, meaning that there is some default .css file somewhere in Wordpress that I did not feel like fighting with.

I did not get notified by Github of this edit. We should definitely give this another shot, as it would be awesome if users could install Monterey right from the blogpost via a single click.

If you want me to take a look I need the login info for that site

**Comment by JeroenVinke** [comment]
This has been completed, and I have added the tiles to the blog
